_sass/navigation.scss

```diff
@@ -68,8 +68,8 @@ nav {
 	font-size: .9rem !important;
 	@media (min-width: 840px) {
 		padding: 0.5rem !important;
-		
-		margin: 0px 3px; 
+
```

Comment:
@amitamrutiya Can you please remove these extra spaces from different parts of the code? it causes lint errors. Else, the code looks good to me